reading_json.py

```diff
@@ -0,0 +1,49 @@
+import os
+import shutil
+import json
+import sys
+
+def main():
+
+    # getting cmd_line_args in format:
+    # "python3 reading_json.py file.json img_dir output_dir"
+    cmd_line_args = sys.argv
+    print(cmd_line_args)
```

Comment:
This line `print(cmd_line_args)` should probably either go away or be more explicit because as a normal user, I have no idea why the program is spitting this out back to me :P